nautobot/extras/tests/test_plugins.py

```diff
@@ -251,6 +242,18 @@ class DummyConfigWithInstalledApps(dummy_config):
         with self.assertRaises(PluginImproperlyConfigured):
             DummyConfigWithInstalledApps.validate({}, settings.VERSION)
 
+    def test_registry_nav_menu_dict(self):
+        """
+        Validate that dummy plugin is adding new items to `registry["nav_menu"]`.
+        """
+        self.assertTrue(registry["nav_menu"]["tabs"].get("Dummy Tab"))
+        self.assertTrue(registry["nav_menu"]["tabs"]["Dummy Tab"]["groups"].get("Dummy Group 1"))
+        self.assertTrue(
+            registry["nav_menu"]["tabs"]["Dummy Tab"]["groups"]["Dummy Group 1"]["items"].get(
+                "plugins:dummy_plugin:dummymodel_list"
+            )
+        )
```

Comment:
Should also test for the inclusion of the new-style menus declared by the plugin as well (dummy circuit group, etc.).

Possibly also verify that the existing (baseline) Circuit menu items are still present after the plugin has augmented the menu.